crates/transaction-pool/src/test_utils/mock.rs

```diff
@@ -21,6 +21,15 @@ pub(crate) type MockTxPool = TxPool<MockOrdering>;
 
 pub type MockValidTx = ValidPoolTransaction<MockTransaction>;
 
+#[cfg(feature = "optimism")]
+use reth_primitives::DEPOSIT_TX_TYPE;
```

Comment:
I think we should consider putting Optimism *only* types into an `optimism` module here